src/github_review_prompts/cli.py

```
@@ -0,0 +1,753 @@
+        print(f"プロンプト抽出: {summary['prompts_extracted']}")
+        print(f"成功率: {summary['success_rate']:.1%}")
+        print(f"処理時間: {summary['processing_time']:.2f}秒")
+
+        if summary['errors']:
+            print(f"\\n⚠️ エラー ({len(summary['errors'])} 件):")
+            for error in summary['errors']:
+                print(f"  - {error}")
+
+    def _generate_review_prompt_with_todos(self, prompts, pr_basic_info, pr_info, no_confirm: bool = False, auto_commit: bool = False) -> str:
+        """レビュープロンプトとTODOリストを生成"""
+        from pathlib import Path
```

Ruff (0.12.2)

Redefinition of unused `Path` from line 8

Remove definition: `Path`

(F811)